docs/reference/ilm/actions/ilm-searchable-snapshot.asciidoc

```diff
@@ -98,7 +101,8 @@ PUT _ilm/policy/my_policy
       "cold": {
         "actions": {
           "searchable_snapshot" : {
-            "snapshot_repository" : "backing_repo"
+            "snapshot_repository" : "backing_repo",
+            "total_shards_per_node" : 2
```

Comment:
I think this is a pretty power-user sort of feature, so I think it'd be best not to have it in the example (I worry about unthinking copy-and-paste)

Reply:
Make sense